training/tensorflow/census/tf-keras/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy>=1.14
+pandas>=0.22
+six>=1.11
+tensorflow>=2
```

Comment:
Can you specify version 2.1.0.
Also double check in the runtime version page the numpy/pandas and six versions are correct